src/cuttlefish.app.src

```diff
@@ -7,5 +7,9 @@
                   kernel,
                   stdlib
                  ]},
-  {env, []}
+  {env, []},
+
+  {maintainers,[]},
+  {licenses,["Apache"]},
+  {links,[{"Github","https://github.com/tsloughter/cuttlefish"}]}
```

Comment:
have this point here or to the basho repo so people find the right repo when clicking on the link in hex?